warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/EventMapper.java

```diff
@@ -106,6 +106,12 @@ public class EventMapper<K1,V1 extends RawRecordContainer,K2,V2> extends StatsDE
      */
     public static final String DISCARD_INTERVAL = "event.discard.interval";
 
+    /**
+     * number which will be used to evaluate whether or not an Event should be processed. If the Event.getEventDate() is less than (now - futureInterval) then
```

Comment:
Shouldn't this read: "...less than or equal to (now + futureInterval)..." or am I missing something?

Reply:
You are correct, I will fix that